### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by pgrimaud**
Hey,

This huge PR will fix : 

- 1 typo in `content/blog/2016-09-18-invisible-field/index.md`
- 1 typo in `content/blog/2017-02-10-keccak/index.md`
- 3 typos in `content/blog/2017-06-17-icos/index.md`
- 1 typo in `content/blog/2017-08-27-the-rise-and-rise-of-social-coins/index.md`
- 1 typo in `content/blog/2017-12-13-toll-equality/index.md`
- 2 typos in `content/blog/2018-05-29-defuse-difficulty-bomb-hard-fork-upgrade/index.md`
- 4 typos in `content/blog/2019-10-06-pow-mining-explicit-social-contract/index.md`
- 1 typo in `content/blog/2020-09-25-core-devs-call-51-attack-solutions/index.md`
- 1 typo in `content/blog/2020-11-27-thanos-hard-fork-upgrade/index.md`
- 1 typo in `content/blog/2022-11-11-the-ethereum-classic-safe-multisignature-wallet-explained/index.md`
- 3 typos in `content/blog/2022-12-07-the-three-pillars-of-ethereum-classic/index.md`
- 1 typo in `content/blog/2022-12-13-ethereum-classic-is-bitcoins-philosophy-with-ethereums-technology/index.md`
- 1 typo in `content/blog/2022-12-21-using-ethereum-classic-with-metamask/index.md`
- 2 typos in `content/development/guides/index.yaml`
- 1 typo in `content/development/testnets/index.yaml`
- 1 typo in `content/faqs/investors/index.yaml`
- 1 typo in `content/faqs/researchers/index.yaml`
- 1 typo in `content/services/apps/apps.collection.yaml`
- 1 typo in `content/services/exchanges/index.yaml`
- 4 typos in `content/videos/videos.collection.en.yaml`

Cheers! :robot:

### CODE REVIEW

File: content/blog/2022-12-13-ethereum-classic-is-bitcoins-philosophy-with-ethereums-technology/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ But Ethereum Classic does not operate like Bitcoin. ETC has the same exact desig
 
 What is an EVM blockchain? 
 
-Another of the Cypherpunks' goals was to integrate [smart contracts](https://en.wikipedia.org/wiki/Smart_contract), or decentralized software programmes, inside the digital currency network. What use is a decentralized currency as Bitcoin if to use it and program the money you need to use apps that are hosted in centralized servers, data centers, or cloud services?
+Another of the Cypherpunks' goals was to integrate [smart contracts](https://en.wikipedia.org/wiki/Smart_contract), or decentralized software programmers, inside the digital currency network. What use is a decentralized currency as Bitcoin if to use it and program the money you need to use apps that are hosted in centralized servers, data centers, or cloud services?
```
Comment (TheCrowbill) [inline content/blog/2022-12-13-ethereum-classic-is-bitcoins-philosophy-with-ethereums-technology/index.md:48]: I think this one is a UK/US spelling choice. The intended word is not "programmers"
```suggestion
Another of the Cypherpunks' goals was to integrate [smart contracts](https://en.wikipedia.org/wiki/Smart_contract), or decentralized software programs, inside the digital currency network. What use is a decentralized currency as Bitcoin if to use it and program the money you need to use apps that are hosted in centralized servers, data centers, or cloud services?
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]
Thank you so much for doing this.

On the requested change, it really doesn't matter which spelling you choose to use. Otherwise, LGTM.

**Review by w1g0 — DISMISSED** [review]
tyvm

File: content/blog/2022-12-13-ethereum-classic-is-bitcoins-philosophy-with-ethereums-technology/index.md (hunk 1 above)
Comment (pgrimaud) [reply content/blog/2022-12-13-ethereum-classic-is-bitcoins-philosophy-with-ethereums-technology/index.md]: You're right, I read it too fast

**Review by w1g0 — APPROVED** [review]

**Review by TheCrowbill — APPROVED** [review]

**Pull request merged**